src/views/Dashboard.vue

```diff
@@ -697,6 +699,8 @@ export default {
       return '-'
     },
     fetchAccount(address) {
+      const conf = this.$http.getSelectedConfig()
+      const decimal = conf.assets.exponent || '6'
```

Comment:
> Thanks for contribution
> 
> `conf.assets.exponent` will always return `undefined`.
> 
> You can not simply use one `exponent` from `conf.assets[x].exponent`, because assets is array. should match the denom.

Ah, okay.  So this should actually be `conf.assets[0].exponent`?
 
> By the way, there's an important task, Do you want to help? add multi language support

Absolutely, how can I help?